### DISCUSSION TITLE
Breaking changes 16.7

### DISCUSSION THREAD

**Discussion by tim-wilkins** [primefaces/community#426] [PrimeNG] [answered]
Is there a list of breaking changes for this?  I've been unable to update to this - one application continually reloads and another hangs on one particular page (there's an error on the console, but the error is about 60 call levels below my code!).  I've just tried using 16.7.2.

**Comment by SoyDiego** [comment]
> Is there a list of breaking changes for this? I've been unable to update to this - one application continually reloads and another hangs on one particular page (there's an error on the console, but the error is about 60 call levels below my code!). I've just tried using 16.7.2.

Unfortunately there isn't a list but you have all the changes here: https://github.com/primefaces/primeng/releases
From which version are you trying to upgrade?

**Comment by tim-wilkins** [comment]
Thanks, I'm currently running 16.6.0 and have updated to the latest Angular 16.  I noticed one or two changes (e.g. to autocomplate events).

**Comment by SoyDiego** [comment]
> Thanks, I'm currently running 16.6.0 and have updated to the latest Angular 16. I noticed one or two changes (e.g. to autocomplate events).

I updated last week to 16.0.2 to 16.7.2 and I didn't have big problems (except the problem with layers and I wrote here with my solution. https://github.com/primefaces/primeng/issues/13757) Maybe this link is useful for you too and with others users with problems.
I don't know why from 16.6.0 to 16.7.2 you have problems.

**Comment by tim-wilkins** [comment]
Looking at console errors (which may or may not be related to the issues I'm having). The hangs seem to be related to model updates and the notification of these through rxjs, probably in the context of a p-table.  The repeated load problem gives an error about an import statement outside a module, but this seems to be in the core loading (and before a "Not alive yet..." so seems unlikely to be the problem.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can check the changelog, we list the breaking changes and other modifications we've made in the changelog.
